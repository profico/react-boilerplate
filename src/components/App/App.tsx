import React from 'react';

import Home from '../../views/Home';

const App: React.FC = () => (
  <div>
    <Home />
  </div>
);

export default App;
