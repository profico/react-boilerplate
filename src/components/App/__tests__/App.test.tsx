import React from 'react';

import { shallow } from 'enzyme';

import App from 'components/App';

describe('App [component] ->', () => {
  it('Should render correctly', () => {
    const output = shallow(<App />);

    expect(output).toMatchSnapshot();
  });
});
