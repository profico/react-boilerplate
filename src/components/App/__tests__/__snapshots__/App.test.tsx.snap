// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App [component] -> Should render correctly 1`] = `
<div>
  <Home />
</div>
`;
